packages/radix-ui-themes/src/components/code.css

```diff
@@ -110,7 +113,8 @@
     color: var(--accent-12);
   }
 
-  :where(.rt-Link) & {
+  :where(.rt-Link) &,
+  &:where(a, button) {
```

Comment:
Since we already have styles for interactive Code, should we support `<Code asChild>` into buttons/links?